Libraries/LibWeb/HTML/Parser/ListOfActiveFormattingElements.cpp

```diff
@@ -112,17 +124,20 @@ Optional<size_t> ListOfActiveFormattingElements::find_index(DOM::Element const&
     return {};
 }
 
-void ListOfActiveFormattingElements::replace(DOM::Element& to_remove, DOM::Element& to_add)
+void ListOfActiveFormattingElements::replace(DOM::Element& to_remove, DOM::Element& to_add, HTMLToken& token)
 {
     for (size_t i = 0; i < m_entries.size(); i++) {
-        if (m_entries[i].element.ptr() == &to_remove)
+        if (m_entries[i].element.ptr() == &to_remove) {
             m_entries[i].element = GC::make_root(to_add);
+            m_entries[i].token = create_own_token(token);
+            break;
+        }
```

Comment:
Looks like we can change this loop into a simple `for (auto& entry : m_entries) { ...`?